models/forms/registration.go

```diff
@@ -0,0 +1,82 @@
+package forms
+
+import (
+	"log"
+	"strings"
+
+	"go-google-scraper-challenge/helpers"
+	"go-google-scraper-challenge/models"
+
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type RegistrationForm struct {
+	Email                string `valid:"Email; Required"`
+	Password             string `valid:"Required; MinSize(6)"`
+	PasswordConfirmation string `valid:"Required; MinSize(6)"`
+}
+
+func init() {
+	lowerCaseErrorMessage := map[string]string{}
+	for key, value := range validation.MessageTmpls {
+		lowerCaseErrorMessage[key] = strings.ToLower(value)
+	}
+
+	validation.SetDefaultMessage(lowerCaseErrorMessage)
```

Comment:
Added this to convert the default error message from the first letter capitalized to lower case, so when combining the error message with the field name instead of `Email Must be a valid email address` it becomes `Email must be a valid email address`